ui/src/kapacitor/components/config/SlackConfigs.tsx

```diff
@@ -73,7 +74,7 @@ class SlackConfigs extends PureComponent<Props, State> {
             />
           )
         })}
-        {isMultipleConfigsSupported && (
+        {this.isAddAllowed && (
```

Comment:
it's not clear to me what `isAddAllowed` would mean, semantically -- would you consider preserving some of the self-awareness around `config`, such as `isConfigAddAllowed`?